packages/edit-site/src/components/actions/index.js

```diff
@@ -47,8 +49,10 @@ export const trashPostAction = {
 						  )
 						: sprintf(
 								// translators: %d: The number of pages (2 or more).
-								__(
-									'Are you sure you want to delete %d pages?'
+								_n(
+									'Are you sure you want to delete %d page?',
```

Comment:
Do we still need this since the number of posts at this point will always be `plural`?

Reply:
**Yes we do!**

Plural forms in various languages around the world are very complex. For example, sometimes the "singular" string for "1" item is also used for 21 or 31.

I recommend reading blog posts such as https://required.com/en/internationalization-localization/ to learn more about this.

Reply:
Is the PR otherwise ready to land?

Reply:
Thanks for sharing, it was valuable. I had also experienced the difference in nodding with Bulgarians 😄 .